traffic_portal/app/src/common/modules/table/serverCapabilityServers/table.assignServersPerCapability.tpl.html

```diff
@@ -0,0 +1,31 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<div class="modal-header">
+    <button type="button" class="close" ng-click="cancel()"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
+    <h3 class="modal-title">Assign Servers to {{::serverCapability.name}}</h3>
+</div>
+<div class="modal-body">
+    <common-grid-controller table-name="serversUnassigned" options="gridOptions"
+                            data="servers" columns="columns" selected-data="selectedServers"></common-grid-controller>
+</div>
+<div class="modal-footer">
+    <button class="btn btn-link" ng-click="cancel()">cancel</button>
+    <button class="btn btn-primary" ng-click="submit()">Submit</button>
```

Comment:
These buttons should have `type="button"` because they don't submit a form (which is the default behavior if you don't specify).

Also "cancel" has different casing than "Submit" - it should probably be capitalized as well.